### ISSUE TITLE
A couple of display issues

### ISSUE THREAD

**Issue by percyb0t** [discord-extensions/essence#32]
Hello! Just wanted to say that this is one of my favorite themes. It's very aesthetically pleasing and I LOVE the purple color palette. I'm not sure how active you are on here -- but there are two bugs I've noticed.

##Icon in user settings
![image](https://github.com/discord-extensions/essence/assets/144847919/49111d44-433d-4e91-ab58-acf0a189cefa)
When you go into user settings, the icon is placed in an odd spot.

##Friends page
![image](https://github.com/discord-extensions/essence/assets/144847919/5d7fa550-15d8-4668-97b7-6d211ae3f1e7)
Not sure if it's supposed to look like this? But the icons of friends are placed in an odd position here as well.

Thank you for the wonderful theme and have a good day! :)

**Comment by LuckFire** [comment]
Yeah it's not supposed to look like that, I do have a rewrite planned (#29) but I'm not entirely sure on when I will legitimately get around to finishing it.

I'm glad you enjoy the theme btw <3
